apps/website/src/examples/Breadcrumbs.extremeTruncation.tsx

```diff
@@ -0,0 +1,36 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import * as React from 'react';
+import { Breadcrumbs, Button, MiddleTextTruncation } from '@itwin/itwinui-react';
+
+export default () => {
+  return (
+    <div
+      style={{
+        border: '1px solid pink',
+        maxWidth: 450,
```

Comment:
There's a bit of overflow on the smallest window width with the current `maxWidth`. Should we change the `maxWidth`?

![image](https://user-images.githubusercontent.com/45748283/215615832-ae866e52-5112-47c2-b006-d2c1dd973a73.png)

Reply:
I am unable to re-create this scenario.  When shrinking the browser window width, I do not see the pink box going outside of the demo frame:
![Screenshot 2023-02-03 at 12 33 59 PM](https://user-images.githubusercontent.com/849817/216669448-06eeed7d-ab6a-45dd-a6e1-9595333d9ad6.png)